### ISSUE TITLE
Dialob filling preview

### ISSUE THREAD

**Issue by KnutResys** [digiexpress-io/digiexpress-parent#184]

**Comment by olev-mutso** [comment]
I do not think this is relevant at all, these 2 different applications, what is the value of showing review in the styles of different application?

**Comment by vahurkrouverk** [comment]
It's about preview, not review?

**Comment by jocelynmutso** [comment]
1. Added support into Eveli (front desk) to display form filling sessions using Gamut(client portal) theme.
2. Gamut theme can be further redefined using tenant configuration if needed.
3. Dialob-related container must be configured to use the following url:

`COMPOSER_FILLING_APP_URL={_eveli_app_}/secured/en/assets/forms`
